paket.dependencies

```diff
@@ -1,13 +1,17 @@
 source https://nuget.org/api/v2
-framework: net45
+source https://api.nuget.org/v3/index.json
 
-nuget IKVM
-nuget Microsoft.NETFramework.ReferenceAssemblies
+nuget IKVM 8.2.0-prerelease0392
 nuget NUnit
-nuget NUnit.ConsoleRunner
+nuget NUnit3TestAdapter
+nuget Microsoft.NET.Test.Sdk
+nuget Microsoft.NETFramework.ReferenceAssemblies
+
+http https://github.com/ikvm-revived/ikvm/releases/download/8.2.0-prerelease.392/IKVM-8.2.0-prerelease.392-tools-net461-any.zip ikvm-8.2.0-prerelease.392-tools-net461-win7-x64/tools.zip
+http https://github.com/ikvm-revived/ikvm/releases/download/8.2.0-prerelease.392/IKVM-8.2.0-prerelease.392-tools-netcoreapp3.1-win7-x64.zip ikvm-8.2.0-prerelease.392-tools-netcoreapp3.1-linux-x64/tools.zip
+http https://github.com/ikvm-revived/ikvm/releases/download/8.2.0-prerelease.392/IKVM-8.2.0-prerelease.392-tools-netcoreapp3.1-linux-x64.zip ikvm-8.2.0-prerelease.392-tools-netcoreapp3.1-win7-x64/tools.zip
 
-http https://sergeytihon.files.wordpress.com/2021/11/ikvmbin-8.1.5717.0.zip
-http https://dlcdn.apache.org/opennlp/opennlp-1.9.4/apache-opennlp-1.9.4-bin.zip
+http https://archive.apache.org/dist/opennlp/opennlp-1.9.1/apache-opennlp-1.9.1-bin.zip
```

Comment:
Why you reverted OpenNLP from `1.9.4` to `1.9.1`?

Reply:
This is done because Lucene 8.2.0 (the version which we have ported our OpenNLP extension from) [targeted 1.9.1](https://github.com/apache/lucene/blob/releases/lucene-solr/8.2.0/lucene/ivy-versions.properties#L181). The most recent version, 9.2.0, also still [targets 1.9.1](https://github.com/apache/lucene/blob/releases/lucene/9.2.0/versions.props#L14).

In theory, it should work with 1.9.4 if they are using semantic versioning. This is just to mitigate any risk where the version makes a difference.